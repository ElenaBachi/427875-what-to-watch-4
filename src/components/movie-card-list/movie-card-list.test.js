import React from "react";
import renderer from "react-test-renderer";
import MovieCardList from "./movie-card-list.jsx";

const films = [
  {
    title: `Pulp Fiction`,
    img: `img/pulp-fiction.jpg`,
  }, {
    title: `No Country for Old Men`,
    img: `img/no-country-for-old-men.jpg`,
  }, {
    title: `Snatch`,
    img: `img/snatch.jpg`,
  }, {
    title: `Moonrise Kingdom`,
    img: `img/moonrise-kingdom.jpg`,
  }, {
    title: `Seven Years in Tibet`,
    img: `img/seven-years-in-tibet.jpg`,
  }, {
    title: `Midnight Special`,
    img: `img/midnight-special.jpg`,
  }, {
    title: `War of the Worlds`,
    img: `img/war-of-the-worlds.jpg`,
  }, {
    title: `Dardjeeling Limited`,
    img: `img/dardjeeling-limited.jpg`,
  }
];

it(`MovieCardList should render correctly`, () => {
  const tree = renderer.create(
      <MovieCardList
        films={films}
      />
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
