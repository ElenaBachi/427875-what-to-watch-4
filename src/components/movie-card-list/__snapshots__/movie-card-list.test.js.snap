// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MovieCardList should render correctly 1`] = `
<div
  className="catalog__movies-list"
>
  <article
    className="small-movie-card catalog__movies-card"
    onClick={[Function]}
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
  >
    <div
      className="small-movie-card__image"
    >
      <video
        autoPlay={false}
        height="175"
        loop={true}
        poster="img-1.jpg"
        width="280"
      />
    </div>
    <h3
      className="small-movie-card__title"
    >
      <a
        className="small-movie-card__link"
        href="movie-page.html"
      >
        title1
      </a>
    </h3>
  </article>
  <article
    className="small-movie-card catalog__movies-card"
    onClick={[Function]}
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
  >
    <div
      className="small-movie-card__image"
    >
      <video
        autoPlay={false}
        height="175"
        loop={true}
        poster="img-2.jpg"
        width="280"
      />
    </div>
    <h3
      className="small-movie-card__title"
    >
      <a
        className="small-movie-card__link"
        href="movie-page.html"
      >
        title2
      </a>
    </h3>
  </article>
  <article
    className="small-movie-card catalog__movies-card"
    onClick={[Function]}
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
  >
    <div
      className="small-movie-card__image"
    >
      <video
        autoPlay={false}
        height="175"
        loop={true}
        poster="img-3.jpg"
        width="280"
      />
    </div>
    <h3
      className="small-movie-card__title"
    >
      <a
        className="small-movie-card__link"
        href="movie-page.html"
      >
        title3
      </a>
    </h3>
  </article>
  ;
</div>
`;
